Add cyrillic languages romanization

itsmeteddyhere opened the pull request:
This adds russian romanization, or rather romanization of the Cyrillic characters used in russian. I think this can be added, but other languages using Cyrillic (for example Ukrainian, Bulgarian) might have slight differences or more characters.

mostafaalagamy commented:
> Still need to add more languages and some more features

Ok I'm waiting for you, take your time!

itsmeteddyhere commented:
I'll be done by the end of the week, I'll let you know

itsmeteddyhere commented:
for the changelog I'd write something like

- moved romanization options into another submenu
- added romanization for a few Cyrillic languages
- added the option to toggle romanization for each song separately

itsmeteddyhere commented:
great, thanks
